lib/Gocdb_Services/APIAuthenticationService.php

```diff
@@ -37,21 +37,19 @@ function __construct() {
      * Returns the APIAuthentication entity associated with the given identifier.
      *
      * @param string $ident Identifier (e.g. X.509 DN as string)
-     * @param string $type  Identifyer type (e.g. "X.509")
      * @return \APIAuthentication APIAuthentication associated with this identifier
      */
-    public function getAPIAuthentication($ident, $type) {
```

Comment:
How come you're removing the type?

Reply:
So that the code doesn't have to determine the type of the incoming identifier. It's sufficent just to look for the identity string.